Before: f85aa0d0452b467e63f363f2e35479e0b3a004618153cef1f87541c6e4d3bec3
After: 905ee37f0a855fb7d1f83563f8c2a6b1b549d8e414b5fc73831290d5d2e533c6
Patch: fe1df2a58ef6aeda0c83c41ac50d743a6857fdcdbebcb747f95d6cfa5d58be62
--- Gemfile.corvid	2012-07-22 20:44:49.398916520 +1000
+++ Gemfile.corvid	2012-07-22 21:27:59.126897063 +1000
@@ -10,20 +10,23 @@
 gem 'rake', group: :rake
 
 group :test do
-  gem 'ci_reporter', require: false
-  gem 'simplecov', require: false
+  gem 'simplecov'
 end if Dir.exists?('test')
 
 group :test_unit do
   gem 'minitest'
-  gem 'guard-minitest', '>= 0.5.0', require: false
+  gem 'guard-minitest', '>= 0.5.0'
 end if Dir.exists?('test/unit')
 
 group :test_spec do
   gem 'rspec'
-  gem 'guard-rspec', require: false
+  gem 'guard-rspec'
 end if Dir.exists?('test/spec')
 
+group :ci do
+  gem 'ci_reporter', require: false
+end
+
 group :doc do
   gem 'yard'
   gem 'rdiscount' # TODO aaaaaaaaaahhhhhhhhhh
--- Guardfile	2012-07-22 20:44:49.402249829 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,44 +0,0 @@
-require 'corvid/guard'
-
-########################################################################################################################
-# test/unit
-
-group :unit do
-  guard 'minitest', test_folders: 'test/unit', test_file_patterns: '*_test.rb' do
-
-    # Src files
-    watch(%r'^lib/(.+)\.rb$') {|m| "test/unit/#{m[1]}_test.rb"}
-
-    # Each test
-    watch(%r'^test/unit/.+_test\.rb$')
-
-    if bulk?
-      # Test stuff affecting everything
-      watch(%r'^test/bootstrap/(?:all|unit).rb') {"test/unit"}
-      watch(%r'^test/helpers/.+$')               {"test/unit"}
-      watch(%r'^test/factories/.+$')             {"test/unit"}
-    end
-  end
-end if Dir.exists?('test/unit')
-
-########################################################################################################################
-# test/spec
-
-rspec_options= read_rspec_options(File.dirname __FILE__)
-group :spec do
-  guard 'rspec', binstubs: true, spec_paths: ['test/spec'], cli: rspec_options, all_on_start: false, all_after_pass: false do
-
-    # Src files
-    watch(%r'^lib/(.+)\.rb$') {|m| "test/spec/#{m[1]}_spec.rb"}
-
-    # Each spec
-    watch(%r'^test/spec/.+_spec\.rb$')
-
-    if bulk?
-      # Test stuff affecting everything
-      watch(%r'^test/bootstrap/(?:all|spec).rb') {"test/spec"}
-      watch(%r'^test/helpers/.+$')               {"test/spec"}
-      watch(%r'^test/factories/.+$')             {"test/spec"}
-    end
-  end
-end if Dir.exists?('test/spec')
